parser/src/parser/grammar.lalrpop

```diff
@@ -262,7 +262,7 @@ BoundaryAtom: Expression = {
     "(" <BoundaryExpr> ")",
     "$" <ident: Identifier> <idx: Index> => Expression::Rand(ident, idx),
     <n: Num_u64> => Expression::Const(n),
-    <binding_access: BindingAccess> => Expression::BindingAccess(binding_access),
+    <binding_access: SymbolAccess> => Expression::SymbolAccess(binding_access),
```

Comment:
nit: Should this be  renamed to `symbol_access`? There are actually a few other instances of `binding_access`. It might be easier to just do search and replace if you decide to do the change.

Reply:
Good point, thanks.